File: Part4/C++/Function_ExtraParameter.cpp
#include <stdio.h>
#include <stdlib.h>

int main()
{
    int extraInput = 5;
    char userInput[ 100 ];
    
    printf( "Type 1 and press enter.\n" );
    fgets( userInput, sizeof( userInput ), stdin );
    
    // Function only takes 1 parameter, but we're passing 2
    int enteredInteger = atoi( userInput, extraInput );
    
    printf( "The user entered the integer %d\n", enteredInteger );
    
    return 0;
}

File: Part4/Python/Function_ExtraParameter.py
def main():
    extra_input = 5
    
    print( "Type 1 and press enter." )
    user_input = input()
    
    # Function only takes 1 parameter, but we're passing 2
    entered_integer = int( user_input, extra_input )
    
    print( f"The user entered the integer { entered_integer }" )

if __name__ == "__main__":
    main()
